tortoise/contrib/pydantic/creator.py

```diff
@@ -344,7 +344,9 @@ def get_submodel(_model: "Type[Model]") -> Optional[Type[PydanticModel]]:
             comment = _cleandoc(func)
             if annotation is not None:
                 pannotations[fname] = annotation
-
+        # Json fields
+        elif field_type is fields.JSONField:
+            pannotations[fname] = Any  # type: ignore
```

Comment:
:thinking: Yes, this sort-of makes sense ito JSON, as it can only support what it can support.